### PR TITLE
Modify dynamic mapping selection for creation and update

### LINKED ISSUE #884: Take the 'dynamic' radio btn out of the Collection create or update page
This radio button is misleading, since `dynamic` is a per-property attribute, not just a global one. I'd rather go for a pre-filled JSON, with a global `"dynamic:" false` property suggested by the app.

### PR DESCRIPTION
**Pull request by jsatge**
## What does this PR do ?

fixes #884

Delete radio buttons for dynamic mapping and add pre-filled JSON, with a global "dynamic:" false property by default.

### CODE REVIEW

File: src/components/Data/Collections/Update.vue [hunk 1]
```diff
@@ -56,9 +54,11 @@ export default {
       )
     },
     mappingAttributes() {
-      return this.collection
-        ? omit(this.collection.mapping, '_kuzzle_info')
+      const mapping = this.collection
+        ? omit(this.collection.mapping, '_kuzzle_info', 'dynamic')
         : null
+      const dynamic = mapping.dynamic
+      return { dynamic: dynamic, properties: mapping }
```
Comment (berthieresteban) [inline src/components/Data/Collections/Update.vue:61]: It seems not possible to get `mapping.dynamic` after omit `dynamic` 
Also, `mapping.dynamic` will lead to an error if `this.collection` is null

File: src/vuex/modules/index/store.ts [hunk 2]
```diff
@@ -183,20 +183,16 @@ const actions = createActions({
...
     collection.mapping = mapping
-    collection.dynamic = dynamic
 
     await rootGetters.kuzzle.$kuzzle.collection.create(index.name, name, {
-      dynamic,
-      properties: {
-        ...mapping
-      }
+      ...mapping
     })
```
Comment (berthieresteban) [inline src/vuex/modules/index/store.ts:189]: can replace `{ ...mapping }` by `mapping`

File: src/vuex/modules/index/store.ts [hunk 3]
```diff
@@ -216,10 +211,7 @@ const actions = createActions({
       collection: name,
       index: index.name,
       body: {
-        dynamic,
-        properties: {
-          ...mapping
-        }
+        ...mapping
       }
```
Comment (berthieresteban) [inline src/vuex/modules/index/store.ts:215]: same here

File: test/e2e/cypress/integration/single-backend/collections.spec.js [hunk 4]
```diff
@@ -93,6 +93,9 @@ describe('Collection management', function() {
       .type('{selectall}{backspace}', { delay: 200, force: true })
       .type(
         `{
+          "dynamic": false,
+        "properties": 
+        {
```
Comment (berthieresteban) [inline test/e2e/cypress/integration/single-backend/collections.spec.js:98]:
```suggestion
"dynamic": false,
"properties": {
```

**Review by berthieresteban — CHANGES_REQUESTED** [review]

**Review by xbill82 — APPROVED** [review]

**Review by rolljee — APPROVED** [review]

**Pull request merged**